src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.controller.exeptions.CartNotFoundException;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "v1/cart/",produces = "application/json")
+@CrossOrigin("*")
+
+public class CartController {
+
+    private CartService cartService;
+    private CartMapper cartMapper;
+
+    public CartController(CartService cartService, CartMapper cartMapper) {
+        this.cartService = cartService;
+        this.cartMapper = cartMapper;
+    }
```

Comment:
proponuję dodawać jedną pustą linię po ostatnim `}` w metodzie

Reply:
Hej, zmieniłem ten kod według zaleceń Kamila bo faktycznie miała to być wstępna implementacja. Nie mam jeszcze mappera i serwizu. Ok zwrócę na to uwagę. Jak coś to od razu mnie punktuj :)